### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by florian-hoenicke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,12 @@
...
-
-- 🔒 **Private data**: Create your own search engine with your own private data
-- 🌐 **Democratization of neural search**: Empowers everyone to use neural search - even people who wouldn't normally have access to AI
-- 🔋 **Batteries included**: simple defaults allow you to provide just the data and get your search use case up and running
+NOW gives the world access to customized neural image search in just one line of code.
+Main features
+- 👶 **Easy**: No engineering skills required
+- ⚡ **Fast**: Set up your search case within minutes
+- 🌈 **quality**: If you provide labels to your documents, jina NOW fine-tunes a model for you
+[//]: # (- : **reliable**: we take care of the deployment and maintainance)
```
Comment (makram93) [inline README.md:20]: Something wrong with this line. Does not render icon and neither it is in bullet point

Comment (florian-hoenicke) [reply README.md:20]: This is a comment in mark-down. It can be re-enabled once we have the wolf deployment.

Comment (makram93) [reply README.md:20]: Ah okay, but I still see the last line on the markdown render.
image

**Comment by CatStark** [comment]
Shouldn't we add a bit more information?
Like a section of requirements (especially since this one tends to crash a lot without enough resources)?

**Comment by florian-hoenicke** [comment]
Good point. I will add the requirements section

**Comment by florian-hoenicke** [comment]
@CatStark I updated the readme

**Review by CatStark — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -12,13 +12,12 @@
...
-
-- 🔒 **Private data**: Create your own search engine with your own private data
-- 🌐 **Democratization of neural search**: Empowers everyone to use neural search - even people who wouldn't normally have access to AI
-- 🔋 **Batteries included**: simple defaults allow you to provide just the data and get your search use case up and running
+NOW gives the world access to customized neural image search in just one line of code.
+Main features
+- 🐥 **Easy**: No engineering skills required
+- 🐎 **Fast**: Set up your search case within minutes
+- 🌈 **quality**: If you provide labels to your documents, jina NOW fine-tunes a model for you
+- 🌳 **reliable**: we take care of the deployment and maintenance (coming soon)
```
Comment (makram93) [inline README.md:20]:
```suggestion
- 🌈 **Quality**: If you provide labels to your documents, Jina NOW fine-tunes a model for you
- 🌳 **Reliable**: We take care of the deployment and maintenance (coming soon)
```

**Pull request merged**